### DISCUSSION TITLE
Mobile app

### DISCUSSION THREAD

**Discussion by erkuysal** [dailydotdev/daily#938] [Feature Requests]
You guys definitely need to make an mobile app. I’ve been searching dev news site and found yours. It seems nice but app would make things lot easier.

**Comment by idoshamun** [comment]
Hi @erkuysal,

I agree with you on this one. Unfortunately, our team is still small, and we can't support the development of an extension, webapp and a mobile app for both platforms. I hope we can pursue a mobile app in the near future.
